docs/API.md

```diff
@@ -444,7 +446,7 @@ Options can be configured as a parameter to the [init()](./interfaces/tracer.htm
 | profiling       | `DD_PROFILING_ENABLED`             | `false`        | Whether to enable profiling. |
 | reportHostname  | `DD_TRACE_REPORT_HOSTNAME`         | `false`        | Whether to report the system's hostname for each trace. When disabled, the hostname of the agent will be used instead. |
 | ingestion.sampleRate | `DD_TRACE_SAMPLE_RATE`        | `-`            | Controls the ingestion sample rate (between 0 and 1) between the agent and the backend. Defaults to deferring the decision to the agent. |
-| ingestion.rateLimit  | `DD_TRACE_RATE_LIMIT`         | `-`            | Controls the ingestion rate limit between the agent and the backend. Defaults to deferring the decision to the agent. |
+| ingestion.rateLimit  | `DD_TRACE_RATE_LIMIT`         | `100`          | Controls the ingestion rate limit between the agent and the backend. Defaults to deferring the decision to the agent. |
```

Comment:
Should this just be removed if the official configuration options is now at the root? Also, is the change consistent with other language?